documentation/sphinx/source/release-notes/release-notes-710.rst

```diff
@@ -4,6 +4,15 @@
 Release Notes
 #############
 
+7.1.67
+======
+* Same as 7.1.66 release with AVX enabled.
+
+7.1.66
+======
+* Released with AVX disabled.
+* Refactor check exclusion logic to reduce work in locality-based exclusions `(PR #11835) <https://github.com/apple/foundationdb/pull/11835>`_
+
```

Comment:
Would you mind adding 7.1.62 to 7.1.65 changes as well?

Reply:
Sounds good, I'll do that once I'm back.